libbeat/cmd/root.go

```diff
@@ -105,7 +105,6 @@ func GenRootCmdWithSettings(beatCreator beat.Creator, settings instance.Settings
 	rootCmd.PersistentFlags().AddGoFlag(flag.CommandLine.Lookup("path.data"))
 	rootCmd.PersistentFlags().AddGoFlag(flag.CommandLine.Lookup("path.logs"))
 	rootCmd.PersistentFlags().AddGoFlag(flag.CommandLine.Lookup("path.home"))
-	rootCmd.PersistentFlags().AddGoFlag(flag.CommandLine.Lookup("system.hostfs"))
```

Comment:
Umm, was this flag used in other beats? (I would be ok with removing this for other beats, it is not so breaking as for Metricbeat).

Reply:
As far as I know, it wasn't.